src/affinder/affinder.py

```diff
@@ -184,10 +193,15 @@ def start_affinder(
         close_affinder.layers.bind(points_layers)
         close_affinder.callback.bind(callback)
 
+        remove_pts_layers.viewer.bind(viewer)
+        remove_pts_layers.layers.bind(points_layers)
```

Comment:
I just realised that it was silly to use magicgui for this at all — functools.partial will do the job:

```python
import functools

...

def remove_layers(viewer, layers):
    for layer in layers:
        viewer.layers.remove(layer)

...

remove_points_layers = functools.partial(
        remove_layers, viewer, [points_layers]
        )

...

else:
    remove_points_layers()
```

I'm pretty sure you can do the same thing with `close_affinder` 🤦, if you want to close #29 with a different PR... 😅